spec/controllers/groups_controller_spec.rb

```diff
@@ -241,6 +241,66 @@
       end
     end
 
+    describe '#create_groups_when_students_work_alone' do
+      # Create students
+      let!(:students) { create_list(:student, 5) }
+
+      context 'assignment.group_max = 1' do
+        let!(:assignment) { create(:assignment) }
+
+        it 'creates groups for individual students' do
+          data = []
+          students.each do |record|
+            student_mapping = [record.user_name, record.user_name]
+            data << student_mapping
+          end
+          ActiveJob::Base.queue_adapter = :test
+
+          get_as instructor, :create_groups_when_students_work_alone,
+                 params: { course_id: course.id, assignment_id: assignment.id },
+                 format: 'js'
+
+          assert_enqueued_with(job: CreateGroupsJob, args: [assignment, data])
+          expect(flash[:error]).to be_blank
+        end
+
+        it 'responds with _poll_job template' do
+          get_as instructor, :create_groups_when_students_work_alone,
+                 params: { course_id: course.id, assignment_id: assignment.id },
+                 format: 'js'
+          expect(response).to render_template('shared/_poll_job')
+        end
+
+        it 'responds with appropriate status' do
+          get_as instructor, :create_groups_when_students_work_alone,
+                 params: { course_id: course.id, assignment_id: assignment.id },
+                 format: 'js'
+          expect(response).to have_http_status(200)
+        end
+      end
+      context 'assignment.group_max > 1' do
```

Comment:
There should be a blank line between each `context` block, and between each `it` block.